openslides/urls.py

```diff
@@ -1,68 +1,13 @@
-from django.conf import settings
 from django.conf.urls import include, patterns, url
 
-from openslides.core.views import ErrorView
-from openslides.utils.plugins import get_urlpatterns
+from openslides.core.views import AngularView
 from openslides.utils.rest_api import router
 
-handler403 = ErrorView.as_view(status_code=403)
```

Comment:
Can we please use one urls.py which contains the new url to the angular index at the right place?